### DISCUSSION TITLE
A quarto book for technical documentations

### DISCUSSION THREAD

**Discussion by giscus** [Damian-Oswald/wiki-test#2] [General]
# A quarto book for technical documentations

https://damian-oswald.github.io/wiki-test/index.html

**Comment by damian-oswald** [comment]
Trying out the giscus commenting function...

**Reply by damian-oswald** [reply]
Let's try some **Markdown** formatting in *this* specific `comment`.

**Comment by AFoletti** [comment]
Test comment
